pkg/policy/l4_filter_test.go

```diff
@@ -80,6 +81,11 @@ func newTestData(logger *slog.Logger) *testData {
 	td.repo.selectorCache = td.sc
 
 	td.wildcardCachedSelector, _ = td.sc.AddIdentitySelector(dummySelectorCacheUser, EmptyStringLabels, api.WildcardEndpointSelector)
+	td.cachedSelectorCIDR = func(cidr api.CIDR) CachedSelector {
+		ess := api.CIDRSlice{cidr}.GetAsEndpointSelectors()
+		cs, _ := td.sc.AddIdentitySelector(dummySelectorCacheUser, EmptyStringLabels, ess[0])
+		return cs
+	}(api.CIDR("10.1.1.1"))
```

Comment:
shouldn't this have a prefix?

Reply:
No, the missing coverage was when the prefix is missing. It is allowed, and we only cought it in the CI rather than unit tests!